lib/pure/ropes.nim

```diff
@@ -130,7 +131,7 @@ proc insertInCache(s: string, tree: Rope): Rope =
       result.left = t
       t.right = nil
 
-proc rope*(s: string): Rope {.rtl, extern: "nro$1Str".} =
+proc rope*(s: string = nil): Rope {.rtl, extern: "nro$1Str".} =
```

Comment:
No, an empty rope is ``nil`` not the string, just remove the default value here.

Reply:
Oh, I added it to be able to create a emty rope.
Because newRope is not exported.